### PR TITLE
Org details view

### PR DESCRIPTION
**Pull request by JMurphyWeb**
#663 

url: `/orgs/1`

TODOS
- [x] i need to ask harry on the following icon colour [issue](https://github.com/emfoundation/ce100-app/issues/663#issuecomment-279764010)
- [x] Check the font sizes/weights when we get access to them https://github.com/emfoundation/ce100-app/issues/669

### CODE REVIEW

**Comment by JMurphyWeb** [comment]
The new functionality is tested in an earlier PR.

**Comment by JMurphyWeb** [comment]
@harrygfox don't merge, i have removed something by mistake

**Comment by JMurphyWeb** [comment]
Ok @harrygfox this is now ready for your love.

**Comment by harrygfox** [comment]
@JMurphyWeb back to you I'm afraid ♻️

**Comment by JMurphyWeb** [comment]
@harrygfox the merge conflicts are fixed. And the function that is not tested in this PR is tested in one of @mk4111s upcoming PRs. So don't worry about the reduced coverage.

File: templates/views/organisations/details.html [hunk 1]
```diff
@@ -16,33 +16,34 @@ <h1 class="title title--with-logo">{{org.name}}</h1>
 
     <div class="card--desktop-small">
       <div class="card card--desktop">
-        <h2 class="card__header card--desktop-header-bottom-border">Our circular economy vision</h2>
+        <h2 class="card__header card--desktop-header-bottom-border">OUR CIRCULAR ECONOMY VISION</h2>
```
Comment (harrygfox) [inline templates/views/organisations/details.html]: the three above elements don't quite make semantic sense.
here you have 2 of the same blocks (with different modifiers) one inside the other and then an element of the block with a block-level modifier.
line `17` is missing a block (card) (or maybe it should be an element like card__wrapper)
line `18` is perfect 
line `19` is a `card__header` but then has a `card` modifier class on it so i'm not sure what's what or what's doing what to look at the markup

File: public/css/main.css [hunk 2]
```diff
@@ -576,11 +588,10 @@ Light-teal:     #66d9e0;
...
     color: initial;
     border: none;
-    background: #002944;
-    background: -webkit-linear-gradient(#002944, #1b4664);
-    background: -o-linear-gradient(#002944, #1b4664);
-    background: -moz-linear-gradient(#002944, #1b4664);
-    background: linear-gradient(#002944, #1b4664);
+    background: #003253;
+}
+.card--header--with-logo {
```
Comment (harrygfox) [inline public/css/main.css]: this is double modifying.
I notice you have a `header` _modifier_ for the `card` block, but also a header _**element**_. 
If this is the latter I'd suggest hyphenating everything:
`card--header-with-logo`.
Remember that this should be used **_in conjunction with_** any other modifiers like `card--header`

Comment (JMurphyWeb) [inline public/css/main.css]: Ohhh interesting.

so `".card .card--header .card--header-with-logo"` ? 

So we can use 2 modifiers ?

File: templates/views/organisations/details.html (hunk 1 above)
Comment (JMurphyWeb) [inline templates/views/organisations/details.html]: @harrygfox https://github.com/emfoundation/ce100-app/issues/700#issuecomment-281310563 we will sort/look into this naming when we do the desktop view.

File: public/css/main.css (hunk 2 above)
Comment (JMurphyWeb) [inline public/css/main.css]: now fixed

**Comment by JMurphyWeb** [comment]
@harrygfox both your comments addressed. Don't worry about reduced coverage - this helper function is tested in a different PR

**Comment by harrygfox** [comment]
@JMurphyWeb This should be merged once conflicts are fixed 👍  (you have my consent to do it yourself rather than sending it back to me)

**Pull request merged**